### DISCUSSION TITLE
[Blog] Ruby 3.4 Documentation: A Step Towards Better Ruby Documentation

### DISCUSSION THREAD

**Discussion by st0012** [ruby/rdoc#1256] [Show and tell]
Since most of the documentation improvements in Ruby 3.4 was achieved through this project, I feel we can celebrate it here too: https://st0012.dev/ruby-3-4-docs

**Comment by mjobin-mdsol** [comment]
Hi, I am just coming back from reading this page https://st0012.dev/ruby-3-4-docs
it was shared to me by a colleague. I want to thank everyone involved in the numerous improvements...

its fun to compare https://docs.ruby-lang.org/en/3.4/ with https://docs.ruby-lang.org/en/3.3/

But as I was excited to share the news with some of my Japanese collegues, and also young teenagers that might want to learn about this language from Japan... I was surprise  to find out the JA docs do not at all follow the same layout ?!?

https://docs.ruby-lang.org/ja/3.3
https://docs.ruby-lang.org/ja/3.4

is that expected? does one need to simply regenerate them? or they need to be updated as well ?

**Reply by st0012** [reply]
Thanks for the kind words 😄
It’s an intentional design from previous maintainers/Ruby committers. Here’s more context: https://st0012.dev/a-rdoc-maintainer-s-view-on-ruby-s-documentation#heading-rdoc-the-tool-the-markup-and-docsruby-langorg
